src/components/Table/PTable.vue

```diff
@@ -12,7 +12,7 @@
               </template>
 
               <template v-for="column in visibleColumns" :key="column">
-                <PTableHeader :style="getColumnStyle(column)">
+                <PTableHeader :style="getColumnStyle(column)" :title="column.label">
```

Comment:
Because of the way these slots are implemented, these title attributes will always work off the passed label and won't display any custom content - I think this is fine but open to pushback if that's not a clear devex.